superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterDivider.tsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { css, SupersetTheme } from '@superset-ui/core';
+import React, { useEffect, useRef, useState } from 'react';
+import Icons from 'src/components/Icons';
+import { Tooltip } from 'src/components/Tooltip';
+
+export interface FilterDividerProps {
+  title: string;
+  description: string;
+  horizontal?: boolean;
+  horizontalOverflow?: boolean;
+}
+
+const useIsTruncated = <T extends HTMLElement>(
```

Comment:
Looks like there's an `isTruncated` hook already in `src/hooks`, but it's for the separate use-case of multiple child elements with element truncation controlled by JS, rather than one element with text truncation controlled by CSS.  I kept the default export of `isTruncated` the element version and added a new export of `useCSSTextTruncation` – does that seem like a good solution?